Edit failover timing doc (#2472)
timescale/docs: use-timescale/ha-replicas/high-availability.md

## Failover

Failover is the process of redirecting traffic from your primary to the HA
replica within 15 seconds of the primary becoming unresponsive. As part of

> failover, the HA replica is promoted to become the new primary. You should not
> notice this process, except for a connection reset. In the background, a new

Reviewer: ```suggestion
failover, the HA replica is promoted to become the new primary and you will notice a connection reset. In the background, a new
```

The new replica is created in a new availability zone to help protect against an
availability zone outage.

## Trigger a switchover

To test the failover mechanism, you can trigger a switchover. A switchover is a
safe operation that attempts a failover, and throws an error if the replica or
primary is not in a state to safely switch.

<Procedure>

### Triggering a switchover

1.  Connect to your primary node as `tsdbadmin` or another user that is part of
    the `tsdbowner` group.

    <Highlight type="note">
    You can also connect to the HA replica and check its node using this procedure.
    </Highlight>

1.  At the `psql` prompt, connect to the `postgres` database:

    ```sql
    \c postgres
    ```

    You should see `postgres=>` prompt.

1.  Check if your instance is currently in recovery:

    ```sql
    select pg_is_in_recovery();
    ```

1.  Check which node is currently your primary:

    ```sql
    select * from pg_stat_replication;
    ```

    Note the `application_name`. This is your service ID followed by the
    node. The important part is the `-an-0` or `-an-1`.

1.  Schedule a switchover:

    ```sql
    CALL tscloud.cluster_switchover();
    ```

>     By default, the switchover occurs in 30s. You can change the time by passing

Author: ```suggestion
    By default, the switchover occurs in 30&nbsp;secs. You can change the time by passing
```

Had to pull out my Chicago guide for this one 😅
